src/Tooltip/mobile-view.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react'
+import ReactMarkdown from 'react-markdown'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+class MobileView extends Component {
+  state = {
+    visible: false
+  }
+
+  openTooltip = e => {
+    e.stopPropagation()
+    this.setState({
+      visible: true
+    })
+  }
+
+  closeTooltip = e => {
+    e.stopPropagation()
+    this.setState({
+      visible: false
+    })
+  }
+
+  render() {
+    return (
+      <>
+        <HighlightedText onClick={this.openTooltip}>
+          {this.props.text}
+        </HighlightedText>
+        {this.state.visible && (
+          <Modal>
+            <ModalBackground onClick={this.closeTooltip}>
+              <ModalContent onClick={this.openTooltip}>
+                <CloseContainer onClick={this.closeTooltip}>
+                  <Line first />
+                  <Line second />
+                </CloseContainer>
+                <Header>{this.props.header}</Header>
+                <ReactMarkdown source={this.props.description} />
+              </ModalContent>
+            </ModalBackground>
+          </Modal>
+        )}
+      </>
+    )
+  }
+}
+
+MobileView.propTypes = {
+  description: PropTypes.string.isRequired,
+  header: PropTypes.string.isRequired,
+  text: PropTypes.node.isRequired
+}
+
+class Modal extends Component {
+  constructor(props) {
```

Comment:
Can we use function component here? There is no state or lifecycle methods in it.

Reply:
Also can we move portal creation logic to separate component? I think we will want to use it for other modals in the future too, and we will probably will rewrite desktop tooltips with portals too.